migrations/20191007173134-add-lastlogin-column-to-users.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  up: function(queryInterface, DataTypes) {
+    return queryInterface.addColumn('Users', 'lastLogin', {
+      type: DataTypes.DATE,
+    });
+  },
+
+  down: function(queryInterface) {
+    return queryInterface.removeColumn('Users', 'lastLogin');
+  },
+};
```

Comment:
Have you considered naming the column `lastLoginAt`?

That would match the convention we have for Date columns.

Reply:
That makes more sense. I'll be sure to update that.